stardoc/stardoc.bzl

```diff
@@ -63,6 +68,17 @@ This rule is an experimental replacement for the existing skylark_doc rule.
 A list of symbol names to generate documentation for. These should correspond to
 the names of rule definitions in the input file. If this list is empty, then
 documentation for all exported rule definitions will be generated.
+""",
+            default = [],
+        ),
+        "semantic_flags": attr.string_list(
+            doc = """
+A list of canonical flags to affect Starlark semantics for the Starlark interpretter
+during documentatoin generation. This should only be used to maintain compatibility with
+non-default semantic flags required to use the given Starlark symbols.
+<br><br>For example, if <code>//foo:bar.bzl</code> does not build except when a user would specify
+<code>--incompatible_foo_semantic=false</code>, then this attribute should contain
```

Comment:
Prefer markdown instead of HTML:
```
`//foo:bar.bzl`
```

Reply:
unfortunately, because the contents of this docstring are put in an HTML table within the generated markdown, we have to continue using HTML tags :\ code quotes within a table cell don't render appropriately as code

Reply:
I think the convention has been to use markdown in docstrings.
It would be good to keep it.

Reply:
This is actually a pretty tricky issue to solve. Lets discuss offline? (I'll file an issue summarizing the results)